interfaces/cos_agent/v0/README.md

```diff
@@ -0,0 +1,118 @@
+# `cos_agent`
+
+## Usage
+
+The `cos_agent` interface loads observability-related file paths and metrics endpoints into relation data.
+
+The first implementation was for the [`grafana-agent`](https://charmhub.io/grafana-agent/libraries/cos_agent) machine charm for "converting" paths to log files, alert rules and dashboards into relation data for the `loki_push_api`, `prometheus_remote_write` and `grafana_dashboards` interfaces.
```

Comment:
While I appreciate the historical note, this is not part of the specification. I'm wondering if we should however make it clear that this is a 'private' interface, not quite meant to be generic and usable by other charms to talk to some observability charms. 
@simskij WDYT?

Reply:
Also we should make it more clear that this is a SUBORDINATE, ergo MACHINE-charm-specific relation. Should these have a special place in charm-relation-interfaces?
I'm actually thinking we might need some special tags and metadata to distinguish them from regular relations, or the interface tester might behave incorrectly. But that's on me, once we implement tests.

Reply:
Yes, I think we should. Let's list it in a separate table in the readme, just like the identity and mlops solution-specific interfaces.

Reply:
Done